scanomatic/io/jsonizer.py

```diff
@@ -245,3 +246,133 @@ def object_hook(obj: dict) -> Union[dict, Enum, Model]:
 
 def loads(s: Union[str, bytes]) -> Any:
     return json.loads(s, object_hook=object_hook)
+
+
+T = TypeVar('T')
+
+
+def copy(o: T) -> T:
```

Comment:
I don't quite understand this typing. Could you elaborate?

Reply:
whatever type you put in you get the same one out.